src/components/Reports/HealthIndicatorReport/HealthIndicatorWidget/HealthIndicatorWidget.graphql

```diff
@@ -0,0 +1,10 @@
+query HealthIndicatorWidget($accountListId: ID!) {
+  healthIndicatorData(accountListId: $accountListId) {
```

Comment:
You could make this a little more efficient by passing in a month so it only loads one item, like I did in #1218.